libs/blocks/carousel/carousel.js

```diff
@@ -321,6 +331,32 @@ function handleChangingSlides(carouselElements) {
   mobileSwipeDetect(carouselElements);
 }
 
+function convertMpcMp4(slides) {
+  slides.forEach((slide) => {
+    const a = slide.querySelector('a');
+    if (!a) return;
+    if (a.href.includes('images-tv.adobe')) {
+      decorateAnchorVideo({
+        src: a.href,
+        anchorTag: a,
+      });
+    }
```

Comment:
Shouldn't this happen automatically with the autoblock logic? ... should this be part of the autoblock logic? I don't know myself, but worth to pose the question and fine if its good as is

Reply:
Autoblocking for mpc.mp4 urls uses an iframe, not video tag. iframes interfere with touch logic and create other problems when in a carousel. Stake holders wanted to be able to use MPC videos or Sharepoint videos. This allows the use of MPC.mp4 video's and converts to the video tag to get around all the problems iframes bring. 

Authors will need to add `#_dnb` to the mpc.mp4 link. Then, this code is activated. Otherwise video autoblock logic works as it normally does.

Reply:
`#_dnb` is the magic here. We basically say - do not autoblock this. We have other plans for it.